contrib/TestHarness/Program.cs.cmake

```diff
@@ -139,9 +139,10 @@ namespace SummarizeTest
                     try
                     {
                         string oldBinaryFolder = (args.Length > 1)  ? args[1] : Path.Combine("/opt", "joshua", "global_data", "oldBinaries");
-                        bool useValgrind = args.Length > 2 && args[2].ToLower() == "true";
-                        int maxTries = (args.Length > 3) ? int.Parse(args[3]) : 3;
-                        return Run(Path.Combine("bin", BINARY), "", "tests", "summary.xml", "error.xml", "tmp", oldBinaryFolder, useValgrind, maxTries, true, Path.Combine("/app", "deploy", "runtime", ".tls_5_1", PLUGIN));
+                        string fdbVersion = (args.Length > 2) ? args[2] : "";

```

Comment:
Well, since we are leaving this TestHarness, or I would guess use System.CommandLine might be better